docs/painless/painless-statements.asciidoc

```diff
@@ -8,7 +8,7 @@ Painless also supports the `for in` syntax from Groovy:
 
 [source,painless]
 ---------------------------------------------------------
-for (item : list) {
```

Comment:
After some discussion with a team member we have decided long-term we will likely deprecate this feature.  If you are still interested would you please change this to still use colon syntax, but in the proper way? ```for (def item : list) {```

Reply:
made the change